clingo/scripts/install_clingo.py

```diff
@@ -1,14 +1,21 @@
+import sys
+
 import archspec.cpu
 
-import spack.bootstrap
+import spack.bootstrap.config
 import spack.main
 
+
+if sys.platform == "linux":
+    CLINGO_BASE_SPEC = "clingo-bootstrap@spack +static_libstdcpp +optimized +ipo ~docs"
+else:
+    CLINGO_BASE_SPEC = "clingo-bootstrap@spack +optimized +ipo ~docs"
```

Comment:
Turns out we can't do this because of the old concretizer: https://github.com/spack/spack-bootstrap-mirrors/actions/runs/6454249960/job/17519405808?pr=24